OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -19,6 +19,11 @@
     <Configurations>Release;Debug;Release-x86</Configurations>
     <ManagedExeLauncher Condition="('$(OS)' != 'Windows_NT')">mono </ManagedExeLauncher>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetPlatform Condition="$([MSBuild]::IsOsPlatform('Windows'))">win-x64</TargetPlatform>
```

Comment:
We still need to support `win-x86` when compiling the 32 bit windows installer, but don't need to expose this for compiling through the IDE.

Reply:
This is only used for making the life easier when building in IDE. When compiling for the 32 bit installer we will use the parameter win-x86.